src/Microsoft.DotNet.ServiceFabric.ServiceHost/Actors/DelegatedActor.cs

```diff
@@ -148,42 +136,13 @@ public override void Intercept(IInvocation invocation)
             base.Intercept(invocation);
         }
 
-        protected override async Task InterceptAsync(IInvocation invocation, Func<Task> call)
```

Comment:
Why did you remove these 2 Intercept implementations? This means that not all methods will be intercepted correctly.

Reply:
Ahh, I see you made AsyncInterceptor handle this. Thats okay I guess. It makes it slightly less performant.

---
In reply to: [406538414](https://github.com/dotnet/arcade-services/pull/1099#discussion_r406538414) [](ancestors = 406538414)

Reply:
Yeah, I kept screwing up some of the implementations (first I forgot to resister stuff, then I forgot to call my new method).

I figured this thing is already doing reflectiony-interceptor magic, a wrapped task or two was worth the simplification.